udata/tests/frontend/test_markdown.py

```diff
@@ -129,11 +129,20 @@ def test_markdown_linkify_relative_with_tooltip(self, app):
             assert el.getAttribute('data-tooltip') == 'Source'
             assert el.firstChild.data == 'foo'
 
-    def test_markdown_not_linkify_mails(self, md2dom):
-        '''Markdown filter should not transform emails to anchors'''
+    def test_markdown_not_linkify_mails(self, md2dom, app):
+        '''Markdown filter should not transform emails to anchors if not specified in config'''
+        allow_mailto = app.config['MD_ALLOW_MAILTO']
```

Comment:
this will only test one value of `MD_ALLOW_MAILTO` (the default one). you should make a test for both values, cf https://github.com/opendatateam/udata/blob/master/udata/core/post/tests/test_frontend.py#L37 for an example.

Reply:
I made two separate tests for both `MD_ALLOW_MAILTO` values based on your tip... tests seems to pass... I let you consider this conversation resolved or not